docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -693,6 +695,227 @@ To handle the renewal, you can use the periodic reloading mechanism:
 %prod.quarkus.http.insecure-requests=disabled
 ----
 
+== Utilizing OpenShift Serving Certificates
```

Comment:
I think the rule is now to avoid capitalizing titles but I see the doc is all like that so probably good for this patch.

Reply:
Fixed everywhere.